fsw/src/to_lab_app.c

```diff
@@ -139,105 +131,96 @@ void TO_Lab_AppMain(void)
         TO_LAB_process_commands();
     }
 
-   CFE_ES_ExitApp(RunStatus);
+    CFE_ES_ExitApp(RunStatus);
 
 } /* End of TO_Lab_AppMain() */
 
 /*
 ** TO delete callback function.
-** This function will be called in the event that the TO app is killed. 
+** This function will be called in the event that the TO app is killed.
 ** It will close the network socket for TO
 */
 void TO_delete_callback(void)
 {
     OS_printf("TO delete callback -- Closing TO Network socket.\n");
-    if ( TO_LAB_Global.downlink_on )
+    if (TO_LAB_Global.downlink_on)
     {
         OS_close(TO_LAB_Global.TLMsockid);
     }
 }
 
-
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 /*                                                                 */
 /* TO_init() -- TO initialization                                  */
 /*                                                                 */
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 void TO_LAB_init(void)
 {
-    int32            status;
-    char             PipeName[16];
-    uint16           PipeDepth;
-    uint16           i;
-    char             ToTlmPipeName[16];
-    uint16           ToTlmPipeDepth;
+    int32  status;
+    char   PipeName[16];
+    uint16 PipeDepth;
+    uint16 i;
+    char   ToTlmPipeName[16];
+    uint16 ToTlmPipeDepth;
 
     CFE_ES_RegisterApp();
     TO_LAB_Global.downlink_on = false;
-    PipeDepth = TO_LAB_CMD_PIPE_DEPTH;
-    strcpy(PipeName,  "TO_LAB_CMD_PIPE");
+    PipeDepth                 = TO_LAB_CMD_PIPE_DEPTH;
+    strcpy(PipeName, "TO_LAB_CMD_PIPE");
     ToTlmPipeDepth = TO_LAB_TLM_PIPE_DEPTH;
-    strcpy(ToTlmPipeName,  "TO_LAB_TLM_PIPE");
+    strcpy(ToTlmPipeName, "TO_LAB_TLM_PIPE");
 
     /*
     ** Register event filter table...
     */
-    CFE_EVS_Register(CFE_TO_EVS_Filters,
-                     sizeof(CFE_TO_EVS_Filters)/sizeof(CFE_EVS_BinFilter_t),
+    CFE_EVS_Register(CFE_TO_EVS_Filters, sizeof(CFE_TO_EVS_Filters) / sizeof(CFE_EVS_BinFilter_t),
```

Comment:
Is it good practice to have an operation inside a function argument?

Reply:
I'm not aware of any rules specifically against it.

Reply:
Side effects are typically something to avoid, but no side effect here.

Reply:
Not really an operation, this is resolved at compile-time.  In practice rather than duplicating the expression it could be preferable to `#define` it and then use that.